minecraft_servers/spigotcraft/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+    "server_name": "spigotcraft",
+    "nice_name": "Spigotcraft",
+    "direct_ip": "spigotcraft.de",
+    "social": {
+      "web": "https://spigotcraft.de/",
+      "twitter": "SpigotCraftDE",
+      "discord": "https://discord.gg/kmrPjPJsup",
+      "teamspeak": "spigotcraft.de",
+      "web_support": "https://spigotcraft.de/Forum/ticketsystem/"
```

Comment:
The support page should be available for everyone (without authentication).
Please use another page (like a landing page) or remove the line.

Reply:
I have removed it